homeassistant/components/advantage_air/sensor.py

```diff
@@ -115,15 +106,13 @@ class AdvantageAirZoneSignal(AdvantageAirEntity, SensorEntity):
 
     _attr_unit_of_measurement = PERCENTAGE
 
-    @property
-    def name(self):
-        """Return the name."""
-        return f'{self._zone["name"]} Signal'
-
-    @property
-    def unique_id(self):
-        """Return a unique id."""
-        return f'{self.coordinator.data["system"]["rid"]}-{self.ac_key}-{self.zone_key}-signal'
+    def __init__(self, instance, ac_key, zone_key):
+        """Initialize an Advantage Air Zone wireless signal sensor."""
+        super().__init__(instance, ac_key, zone_key)
+        self._attr_name = f'{self._zone["name"]} Signal'
```

Comment:
This appears to prevent the name from being updated by the source without a restart.